CControl_Handler.hh

```diff
@@ -24,11 +24,13 @@ public:
 private:
 
   V2718 *fV2718;
-  V1495 *fV1495;
   DDC10 *fDDC10;
+  V1495 *fV1495;
 
   int fStatus;
   int fCurrentRun;
+  int fBID;
+  int fBoardHandle;
```

Comment:
Is there a need to make these members rather than locals? I don't see them used anywhere else.

Reply:
> Is there a need to make these members rather than locals? I don't see them used anywhere else.

Not really. I guess they are just leftovers from some other variation of the code.